src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -53,7 +53,7 @@
 
 /**
  * Simple docker client for Pipeline.
- * 
+ *
```

Comment:
Please avoid making extraneous diff hunks, particular whitespace changes. You should be able to configure popular editors to not touch lines of code unless and until you edit them.